src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenRefLocalTests.cs

```diff
@@ -3120,5 +3120,27 @@ public void F()
                 Diagnostic(ErrorCode.ERR_ArrayInitToNonArrayType, "{7,8,9}").WithLocation(14, 28)
             );
         }
+
+        [Fact, WorkItem(25264, "https://github.com/dotnet/roslyn/issues/25264")]
+        public void TestNewRefArray()
```

Comment:
>TestNewRefArray [](start = 20, length = 15)

We should test SemanticModel and IOperation, including scenarios when there is an object initializer and a collection initializer.